src/components/CardBrand.tsx

```diff
@@ -0,0 +1,43 @@
+import { useEffect } from 'react';
+import styled from 'styled-components';
+import Visa from '../asset/Visa.svg';
+import MasterCard from '../asset/Mastercard.svg';
+import { useState } from 'react';
+import REGEX from '../constants/regex';
+
+const Image = styled.img`
+  width: 36px;
+  height: 28px;
+`;
+type CardBrand = 'Visa' | 'MasterCard';
+
+const CardBrand = ({ ...props }) => {
+  const { firstCardNumbers } = props;
+  const BRAND_TABLE: Record<CardBrand, string> = {
+    Visa: Visa,
+    MasterCard: MasterCard,
+  };
+
+  const [brand, setBrand] = useState<CardBrand>('Visa');
+  const [isBrand, setIsBrand] = useState(false);
```

Comment:
isBrand는 brand에 의존하는 상태이기 때문에 꼭 별도의 상태로 관리될 필요는 없어보여요.
```suggestion
  const [brand, setBrand] = useState<CardBrand | null>(null);
  const isBrand = brand !== null
```

Reply:
위에서 작성한 commit과 함께 반영 완료했습니다 :)

https://github.com/woowacourse/react-payments/commit/dd89649f039270dede3b41e2bb5ca946fa9ccdf0